### DISCUSSION TITLE
Wazuh - Unable to tuning rules when following the SO guide

### DISCUSSION THREAD

**Discussion by wildlyunder** [Security-Onion-Solutions/securityonion#7047] [Unsupported Versions]
Hi,

I've followed the following guide: [Tuning Rules](https://docs.securityonion.net/en/2.3/wazuh.html#tuning-rules) for Wazuh to suppress  rules via `/opt/so/rules/hids/local_rules.xml`

I've event copied the following:
```
<group name="customrules,">
  <rule id="60106" level="3" noalert="1" overwrite="yes">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^528$|^540$|^673$|^4624$|^4769$</field>
    <description>Windows Logon Success</description>
    <options>no_full_log</options>
    <mitre>
      <id>T1078</id>
    </mitre>
    <group>authentication_success,pci_dss_10.2.5,gpg13_7.1,gpg13_7.2,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

  <rule id="60137" level="3" noalert="1" overwrite="yes">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^538$|^551$|^4634$|^4647$</field>
    <description>Windows User Logoff</description>
    <options>no_full_log</options>
    <group>pci_dss_10.2.5,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
</group>
```

Then I've restarted Wazuh `sudo so-wazuh-restart` and I've waiting 4 hrs and these alerts are still coming in @250 per minute.

Any ideas why the rules aren't suppressed?

**Comment by IzacFrank** [comment]
Hi,

I don't know for Wazuh but for other tools, you need to execute "sudo so-rule-update" after restarting it.

I also always execute "sudo salt '*' state.highstate" to check, if there is anything wrong before updating rules.

I hope it helps.

**Reply by wildlyunder** [reply]
@IzacFrank Thanks for that idea, but it didn't work.... it did give me an idea to look on the FWD Nodes to see if the updated `/opt/so/rules/hids/local_rules.xml` file was pushed to them.. it wasn't.

Do I need to manually update `/opt/so/rules/hids/local_rules.xml` on each FWD node manually?

**_Update: I can confirm if I manually update `/opt/so/rules/hids/local_rules.xml` file on each of the FWD node the suppression works... can I do this centrally from the Master via `/opt/so/rules/hids/local_rules.xml` and push this out?_**

**Comment by weslambert** [comment]
I'd have to double-check but IIRC, Wazuh rules are currently managed on a per-node basis, given that each node run it's own Wazuh manager that could technically have separate agents reporting to it.

**Reply by wildlyunder** [reply]
> @weslambert Any plans to enable updates to `/opt/so/rules/hids/local_rules.xml` on all assets including FWD nodes using salt?
